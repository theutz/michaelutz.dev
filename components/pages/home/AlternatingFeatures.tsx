/* eslint-disable @next/next/no-img-element */
import Image from "next/image";
import { ThumbUpIcon, GlobeIcon } from "@heroicons/react/outline";
import {
  useEndorsement,
  Endorsement as EndorsementType,
} from "hooks/useEndorsement";
import { useState } from "react";
import { classNames } from "utils/classNames";

function Endorsement({
  name,
  title,
  company,
  content,
  image,
}: EndorsementType) {
  const [showFull, setShowFull] = useState(false);
  return (
    <blockquote>
      <div onClick={() => setShowFull((prev) => !prev)}>
        <p
          className={classNames(
            "text-base text-gray-500 block whitespace-pre-line cursor-pointer",
            !showFull && "line-clamp-3"
          )}
        >
          &ldquo;{content.trim()}&rdquo;
        </p>
      </div>
      <footer className="mt-3">
        <div className="flex items-center space-x-3">
          <div className="relative flex-shrink-0">
            <div className="w-6 h-6">
              <Image
                className="rounded-full"
                layout="fill"
                objectFit="contain"
                src={image}
                alt={name}
              />
            </div>
          </div>
          <div className="text-base font-medium text-gray-700">
            {name}, {title} @ {company}
          </div>
        </div>
      </footer>
    </blockquote>
  );
}

export function AlternatingFeatureSections() {
  const { endorsement, increment } = useEndorsement();
  return (
    <div className="relative pt-16 pb-32 overflow-hidden">
      <div
        aria-hidden="true"
        className="absolute inset-x-0 top-0 h-48 bg-gradient-to-b from-gray-100"
      />
      <div className="relative">
        <div className="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24">
          <div className="max-w-xl px-4 mx-auto sm:px-6 lg:py-16 lg:max-w-none lg:mx-0 lg:px-0">
            <div>
              <div>
                <span className="flex items-center justify-center w-12 h-12 rounded-md bg-gradient-to-r from-purple-600 to-indigo-600">
                  <ThumbUpIcon
                    className="w-6 h-6 text-white"
                    aria-hidden="true"
                  />
                </span>
              </div>
              <div className="mt-6">
                <h2 className="text-3xl font-extrabold tracking-tight text-gray-900">
                  The Humans Have Spoken
                </h2>
                <p className="mt-4 text-lg text-gray-500">
                  Building tech products is also about building relationships.
                  Here&apos;s what some of my co-workers have to say about
                  working with me.
                </p>
                <div className="mt-6">
                  <button
                    onClick={() => increment()}
                    className="flex inline-flex px-4 py-2 text-base font-medium text-white border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border rounded-md shadow-sm hover:from-purple-700 hover:to-indigo-700"
                  >
                    Read Another
                  </button>
                </div>
              </div>
            </div>
            <div className="pt-6 mt-8 border-t border-gray-200">
              <Endorsement {...endorsement} />
            </div>
          </div>
          <div className="mt-12 sm:mt-16 lg:mt-0">
            <div className="pl-4 -mr-48 sm:pl-6 md:-mr-16 lg:px-0 lg:m-0 lg:relative lg:h-full">
              <div className="relative lg:m-0 lg:h-full">
                <img
                  className="w-full shadow-xl rounded-xl ring-1 ring-black ring-opacity-5 lg:absolute lg:left-0 lg:h-full lg:w-auto lg:max-w-none filter brightness-90"
                  src="/assets/images/seattle-skyline.jpg"
                  alt="Inbox user interface"
                />
                <div className="absolute inset-0 mix-blend-overlay bg-gradient-to-r from-purple-800 to-indigo-600 lg:w-massive"></div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div className="mt-24">
        <div className="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24">
          <div className="max-w-xl px-4 mx-auto sm:px-6 lg:py-32 lg:max-w-none lg:mx-0 lg:px-0 lg:col-start-2">
            <div>
              <div>
                <span className="flex items-center justify-center w-12 h-12 rounded-md bg-gradient-to-r from-purple-600 to-indigo-600">
                  <GlobeIcon
                    className="w-6 h-6 text-white"
                    aria-hidden="true"
                  />
                </span>
              </div>
              <div className="mt-6">
                <h2 className="text-3xl font-extrabold tracking-tight text-gray-900">
                  Better understand your developer
                </h2>
                <p className="mt-4 text-lg text-gray-500">
                  I was born in lovely Seattle, Washington in the USA. I met my
                  wife in Nashville, Tennessee, and we&apos;ve moved to
                  Helsinki, Finland to start a family.
                </p>
                <div className="mt-6">
                  <a
                    href="#"
                    className="inline-flex px-4 py-2 text-base font-medium text-white border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border rounded-md shadow-sm hover:from-purple-700 hover:to-indigo-700"
                  >
                    Get started
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div className="mt-12 sm:mt-16 lg:mt-0 lg:col-start-1">
            <div className="pr-4 -ml-48 sm:pr-6 md:-ml-16 lg:px-0 lg:m-0 lg:relative lg:h-full">
              <div className="relative lg:h-full lg:m-0">
                <img
                  className="w-full shadow-xl rounded-xl ring-1 ring-black ring-opacity-5 lg:absolute lg:right-0 lg:h-full lg:w-auto lg:max-w-none"
                  src="/assets/images/helsinki-skyline.jpg"
                  alt="Customer profile user interface"
                />
                <div className="absolute inset-0 lg:left-auto mix-blend-overlay bg-gradient-to-r from-purple-800 to-indigo-600 lg:w-massive"></div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}

export default AlternatingFeatureSections;
