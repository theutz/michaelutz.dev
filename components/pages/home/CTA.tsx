export function CTA() {
  return (
    <div className="bg-white">
      <div className="max-w-4xl px-4 py-16 mx-auto sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8 lg:flex lg:items-center lg:justify-between">
        <h2 className="text-4xl font-extrabold tracking-tight text-gray-900 sm:text-4xl">
          <span className="block">Ready to get started?</span>
          <span className="block text-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-clip-text">
            Get in touch or create an account.
          </span>
        </h2>
        <div className="mt-6 space-y-4 sm:space-y-0 sm:flex sm:space-x-5">
          <a
            href="#"
            className="flex items-center justify-center px-4 py-3 text-base font-medium text-white border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border rounded-md shadow-sm hover:from-purple-700 hover:to-indigo-700"
          >
            Learn more
          </a>
          <a
            href="#"
            className="flex items-center justify-center px-4 py-3 text-base font-medium text-indigo-800 border border-transparent rounded-md shadow-sm bg-indigo-50 hover:bg-indigo-100"
          >
            Get started
          </a>
        </div>
      </div>
    </div>
  );
}

export default CTA;
