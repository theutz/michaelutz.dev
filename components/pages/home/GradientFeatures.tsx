import {
  BeakerIcon,
  ChatIcon,
  DatabaseIcon,
  DocumentDuplicateIcon,
  HomeIcon,
  QuestionMarkCircleIcon,
  SparklesIcon,
  TerminalIcon,
} from "@heroicons/react/outline"

const features = [
  {
    name: "The Basics",
    description:
      "HTML, CSS, and JavaScript. The world runs on this stuff, and I've been using it for decades.",
    icon: HomeIcon,
  },
  {
    name: "Front-End Frameworks",
    description:
      "Tons of experience with React, and some experience with Angular and Vue. I'd love to learn Svelte.",
    icon: SparklesIcon,
  },
  {
    name: "Speaking of React...",
    description:
      "Familiar with all the old favorites like Redux & Enzyme, as well as all the newest goodies.",
    icon: ChatIcon,
  },
  {
    name: "What About The Backend?",
    description:
      "I can write a mean backend (pun intended), but I like PostgreSQL and TypeScript for sanity.",
    icon: DatabaseIcon,
  },
  {
    name: "Can you do WordPress?",
    description:
      "Sure! I've been writing WordPress sites since 2008 or so. PHP & MySQL are my friends, too.",
    icon: DocumentDuplicateIcon,
  },
  {
    name: "Can you write tests?",
    description:
      "I love writing tests! Unit testing with Jest and integration/end-to-end testing with Cypress.",
    icon: BeakerIcon,
  },
  {
    name: "So, you just do tech stuff?",
    description:
      "Not at all! I have a passion and eye for design, user experience, and accessibility.",
    icon: TerminalIcon,
  },
  {
    name: "How did you make this site?",
    description:
      "I work smarter not harder. This site is made and deployed with Tailwind CSS & UI, NextJS, and Vercel.",
    icon: QuestionMarkCircleIcon,
  },
]

export function GradientFeatures() {
  return (
    <div className="bg-gradient-to-r from-purple-800 to-indigo-700">
      <div className="max-w-4xl px-4 py-16 mx-auto sm:px-6 sm:pt-20 sm:pb-24 lg:max-w-7xl lg:pt-24 lg:px-8">
        <h2 className="text-3xl font-extrabold tracking-tight text-white">
          Skills you&apos;ll definitely want
        </h2>
        <p className="max-w-3xl mt-4 text-lg text-purple-200">
          I&apos;ve been making web stuff for decades. I&apos;ve got experience
          with multiple content management systems, and I love the new JAMstack
          stuff!
        </p>
        <div className="mt-12 grid grid-cols-1 gap-x-6 gap-y-12 sm:grid-cols-2 lg:mt-16 lg:grid-cols-4 lg:gap-x-8 lg:gap-y-16">
          {features.map((feature) => (
            <div key={feature.name}>
              <div>
                <span className="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10">
                  <feature.icon
                    className="w-6 h-6 text-white"
                    aria-hidden="true"
                  />
                </span>
              </div>
              <div className="mt-6">
                <h3 className="text-lg font-medium text-white">
                  {feature.name}
                </h3>
                <p className="mt-2 text-base text-purple-200">
                  {feature.description}
                </p>
              </div>
            </div>
          ))}
        </div>
      </div>
    </div>
  )
}

export default GradientFeatures
