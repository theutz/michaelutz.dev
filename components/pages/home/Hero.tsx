/* eslint-disable @next/next/no-img-element */

export function Hero() {
  return (
    <div className="relative">
      <div className="absolute inset-x-0 bottom-0 bg-gray-100 h-1/2" />
      <div className="mx-auto max-w-7xl sm:px-6 lg:px-8">
        <div className="relative shadow-xl sm:rounded-2xl sm:overflow-hidden">
          <div className="absolute inset-0">
            <img
              className="object-cover w-full h-full"
              src="https://images.unsplash.com/photo-1521737852567-6949f3f9f2b5?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2830&q=80&sat=-100"
              alt="People working on laptops"
            />
            <div className="absolute inset-0 bg-gradient-to-r from-purple-800 to-indigo-700 mix-blend-multiply" />
          </div>
          <div className="relative px-4 py-16 sm:px-6 sm:py-24 lg:py-32 lg:px-8">
            <h1 className="text-4xl font-extrabold tracking-tight text-center sm:text-5xl lg:text-6xl">
              <span className="block text-white">Not Your Average</span>
              <span className="block text-indigo-200">Web Dude</span>
            </h1>
            <p className="max-w-lg mx-auto mt-6 text-xl text-center text-indigo-200 sm:max-w-3xl">
              I&apos;m passionate about creating experiences for the web that
              are useful, enjoyable, and intuitive.
            </p>
            <div className="max-w-sm mx-auto mt-10 sm:max-w-none sm:flex sm:justify-center">
              <div className="space-y-4 sm:space-y-0 sm:mx-auto sm:inline-grid sm:grid-cols-2 sm:gap-5">
                <a
                  href="#"
                  className="flex items-center justify-center px-4 py-3 text-base font-medium text-indigo-700 bg-white border border-transparent rounded-md shadow-sm hover:bg-indigo-50 sm:px-8"
                >
                  Learn More
                </a>
                <a
                  href="#"
                  className="flex items-center justify-center px-4 py-3 text-base font-medium text-white bg-indigo-500 border border-transparent rounded-md shadow-sm bg-opacity-60 hover:bg-opacity-70 sm:px-8"
                >
                  Reach Out
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}

export default Hero;
