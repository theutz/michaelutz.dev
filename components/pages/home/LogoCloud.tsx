/* eslint-disable @next/next/no-img-element */
import Image, { ImageProps } from "next/image";
import unityLogo from "public/assets/images/unity-logo.png";
import smartlyLogo from "public/assets/images/smartly-logo.png";
import quenticLogo from "public/assets/images/quentic-logo.png";
import uhsLogo from "public/assets/images/uhs-logo.png";
import ersLogo from "public/assets/images/ers-logo.png";
import { classNames } from "utils/classNames";

export const companies: readonly Pick<ItemProps, "src" | "alt">[] = [
  { src: unityLogo, alt: "Unity" },
  { src: smartlyLogo, alt: "Smartly.io" },
  { src: quenticLogo, alt: "Quentic" },
  { src: uhsLogo, alt: "Universal Hospital Systems" },
  { src: ersLogo, alt: "Emergency Reporting" },
];

export function LogoCloud() {
  return (
    <div className="bg-gray-100" data-cy="logo-cloud">
      <div className="px-4 py-16 mx-auto max-w-7xl sm:px-6 lg:px-8">
        <p className="text-sm font-semibold tracking-wide text-center text-gray-500 uppercase">
          Trusted by a handful of former employers
        </p>
        <div
          data-cy="logo-cloud-container"
          className="mt-6 grid grid-cols-2 gap-8 md:grid-cols-6 lg:grid-cols-5"
        >
          {companies.map((company, i) => (
            <Item
              key={company.alt}
              {...company}
              current={i + 1}
              length={companies.length}
            />
          ))}
        </div>
      </div>
    </div>
  );
}

type ItemProps = {
  src: Exclude<ImageProps["src"], string>;
  alt: ImageProps["alt"];
  current: number;
  length: number;
};

function Item({ src, alt, current, length }: ItemProps) {
  const isLast = current === length;
  const is2ndToLast = length - 1 === current;

  return (
    <div
      className={classNames(
        "relative filter grayscale opacity-40 h-12 md:col-span-2 flex justify-center lg:col-span-1",
        isLast ? "col-span-2 md:col-start-4" : "col-span-1",
        is2ndToLast && "md:col-start-2"
      )}
    >
      <Image layout="fill" objectFit="contain" src={src} alt={alt} />
    </div>
  );
}

export default LogoCloud;
