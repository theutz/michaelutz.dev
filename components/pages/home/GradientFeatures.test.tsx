import { render, screen } from "@testing-library/react";
import { GradientFeatures } from "components/pages/home/GradientFeatures";

describe("<GradientFeatures />", () => {
  it("renders", () => {
    // Arrange

    // Act
    const { asFragment } = render(<GradientFeatures />);

    // Assert
    expect(asFragment()).toMatchSnapshot();
  });
});
