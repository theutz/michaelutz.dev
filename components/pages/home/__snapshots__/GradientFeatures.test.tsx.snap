// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GradientFeatures /> renders 1`] = `
<DocumentFragment>
  <div
    class="bg-gradient-to-r from-purple-800 to-indigo-700"
  >
    <div
      class="max-w-4xl px-4 py-16 mx-auto sm:px-6 sm:pt-20 sm:pb-24 lg:max-w-7xl lg:pt-24 lg:px-8"
    >
      <h2
        class="text-3xl font-extrabold tracking-tight text-white"
      >
        Skills you'll definitely want
      </h2>
      <p
        class="max-w-3xl mt-4 text-lg text-purple-200"
      >
        I've been making web stuff for decades. I've got experience with multiple content management systems, and I love the new JAMstack stuff!
      </p>
      <div
        class="mt-12 grid grid-cols-1 gap-x-6 gap-y-12 sm:grid-cols-2 lg:mt-16 lg:grid-cols-4 lg:gap-x-8 lg:gap-y-16"
      >
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              The Basics
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              HTML, CSS, and JavaScript. The world runs on this stuff, and I've been using it for decades.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 3v4M3 5h4M6 17v4m-2-2h4m5-16l2.286 6.857L21 12l-5.714 2.143L13 21l-2.286-6.857L5 12l5.714-2.143L13 3z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Front-End Frameworks
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Tons of experience with React, and some experience with Angular and Vue. I'd love to learn Svelte.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 12h.01M12 12h.01M16 12h.01M21 12c0 4.418-4.03 8-9 8a9.863 9.863 0 01-4.255-.949L3 20l1.395-3.72C3.512 15.042 3 13.574 3 12c0-4.418 4.03-8 9-8s9 3.582 9 8z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Speaking of React...
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Familiar with all the old favorites like Redux & Enzyme, as well as all the newest goodies.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M4 7v10c0 2.21 3.582 4 8 4s8-1.79 8-4V7M4 7c0 2.21 3.582 4 8 4s8-1.79 8-4M4 7c0-2.21 3.582-4 8-4s8 1.79 8 4m0 5c0 2.21-3.582 4-8 4s-8-1.79-8-4"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              What About The Backend?
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              I can write a mean backend (pun intended), but I like PostgreSQL and TypeScript for sanity.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 7v8a2 2 0 002 2h6M8 7V5a2 2 0 012-2h4.586a1 1 0 01.707.293l4.414 4.414a1 1 0 01.293.707V15a2 2 0 01-2 2h-2M8 7H6a2 2 0 00-2 2v10a2 2 0 002 2h8a2 2 0 002-2v-2"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Can you do WordPress?
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Sure! I've been writing WordPress sites since 2008 or so. PHP & MySQL are my friends, too.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19.428 15.428a2 2 0 00-1.022-.547l-2.387-.477a6 6 0 00-3.86.517l-.318.158a6 6 0 01-3.86.517L6.05 15.21a2 2 0 00-1.806.547M8 4h8l-1 1v5.172a2 2 0 00.586 1.414l5 5c1.26 1.26.367 3.414-1.415 3.414H4.828c-1.782 0-2.674-2.154-1.414-3.414l5-5A2 2 0 009 10.172V5L8 4z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Can you write tests?
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              I love writing tests! Unit testing with Jest and integration/end-to-end testing with Cypress.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 9l3 3-3 3m5 0h3M5 20h14a2 2 0 002-2V6a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              So, you just do tech stuff?
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Not at all! I have a passion and eye for design, user experience, and accessibility.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8.228 9c.549-1.165 2.03-2 3.772-2 2.21 0 4 1.343 4 3 0 1.4-1.278 2.575-3.006 2.907-.542.104-.994.54-.994 1.093m0 3h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              How did you make this site?
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              I work smarter not harder. This site is made and deployed with Tailwind CSS & UI, NextJS, and Vercel.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
